CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-### 1.0.2 (Next)
+### 1.1.0 (Next)
 
+* [#78](https://github.com/dblock/strava-ruby-client/pull/78): Drops supported endpoint/method `activity_photos` - [@simonneutert](https://github.com/simonneutert).
```

Comment:
Can I nitpick one more? The method is no longer supported. So it's "Drops unsupported endpoint/method"?